spec/acceptance/rundeck_spec.rb

```diff
@@ -7,14 +7,16 @@
       class { 'java':
         distribution => 'jre'
       }
-      class { 'rundeck': }
+      class { 'rundeck':
+        package_ensure => '2.9.4'
```

Comment:
Why version 2.9.4 if I may ask ?

Reply:
That was the latest version of the 2.x branch that I could find on their mirrors.

Reply:
I'd say 2.11.5 is the latest 2.x from [the release page](http://rundeck.org/news/2018/07/06/rundeck-2.11.5.html) and available on [their repo](https://rundeck.bintray.com/rundeck-rpm/:rundeck-2.11.5-1.56.GA.noarch.rpm). I guess we should try to run the tests against the latest 2.x, although 2.9.4 is probably fine :)

Reply:
ups sorry. I will update it later.